OasisDEX/oasis-borrow
# batch cache feature toggle uses env var

# [Testing batch cache](https://app.shortcut.com/oazo-apps/story/5377/test-off-chain-cache)
  
## Changes 👷‍♀️

Includes env vars for infura cache so we can configure without a deploy.
- `INFURA_CACHE_ENABLED` enables for everyone.
- `INFURA_CACHE_TTL` configures cache expiry time.
  
## How to test 🧪
- Missing env vars should cause the cache to be bypassed (check network tab for `/api/infuraCallsCache` - if this is present in network tab traffic has been routed through the cache)
- `INFURA_CACHE_ENABLED=false` and `BatchCache` enabled in `Application tab -> features localstorage` should cause traffic to be routed through `/api/infuraCallsCache`.
- Ensure TTL can be reduced and increased (probably requires service restart).  You can check number of keys from oasis-borrow runtime output.
